metadata-models/src/main/pegasus/com/linkedin/datajob/EditableDatajobProperties.pdl

```diff
@@ -10,5 +10,12 @@ import com.linkedin.common.ChangeAuditStamps
   "name": "editableDatajobProperties"
 }
 record EditableDatajobProperties includes ChangeAuditStamps {
```

Comment:
Can we also rename this to EditableDataJobProperties? Didn't catch this before.

Reply:
@jjoyce0510 done!